BeatleyTQD/inVision
# Why delete works

# Description
Users are now able to delete Whys
Fixes #13 
## Type of change
# Testing Instructions
Navigate to your list of Whys
Select one to be deleted
Contemplate deeply why you have decided this Why is no longer appropriate.
If you are certain, confirm.
If not, cancel.
# Checklist:
